cocotb/share/include/cocotb_utils.h

```diff
@@ -42,8 +44,26 @@ extern void* utils_dyn_sym(void *handle, const char* sym_name);
 
 extern int is_python_context;
 
-void to_python(void);
-void to_simulator(void);
+// to_python and to_simulator are implemented as macros functions so that the
+// logs reference the user's lineno and filename
```

Comment:
```suggestion
// to_python and to_simulator are implemented as macros instead of functions so
// that the logs reference the user's lineno and filename
```